compiler/incremental-compilation-impl/src/org/jetbrains/kotlin/incremental/classpathDiff/ClasspathChangesComputer.kt

```diff
@@ -207,10 +227,18 @@ object ClasspathChangesComputer {
         currentClassSnapshots: List<AccessibleClassSnapshot>,
         previousClassSnapshots: List<AccessibleClassSnapshot>
     ): ProgramSymbolSet {
-        // DirtyData.dirtyLookupSymbols may contain added symbols (they can also impact recompilation -- see examples in
-        // JavaClassChangesComputer), so we need to consider all classes on both the current and previous classpath. It's okay for the
-        // combined list to contain overlapping ClassIds.
-        val allClasses = (currentClassSnapshots.asSequence() + previousClassSnapshots.asSequence()).asIterable()
+        // Convert dirtyLookupSymbols to ProgramSymbols.
```

Comment:
nit: Renaming this method to `toProgramSymbols` increases readability.

Also, I found the previous comment much more concise and readable.

Reply:
> nit: Renaming this method to toProgramSymbols increases readability.

Done 

> Also, I found the previous comment much more concise and readable.

Right, it hurt my ear when I read it again. Reverted it.